### PR TITLE
build(*): replace npm with yarn

### PR DESCRIPTION
**Pull request by garrappachc**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -16,13 +16,14 @@ ENV NODE_ENV=${NODE_ENV}
 
 COPY package*.json ./
 
-RUN npm install --only=production
+RUN yarn install \
```
Comment (c43721) [inline Dockerfile:20]: To install prod deps, do `yarn install --prod`

**Review by c43721 — CHANGES_REQUESTED** [review]
let's keep our prod deps :)

**Comment by c43721** [comment]
LGTM

File: Dockerfile (hunk 1 above)
Comment (c43721) [reply Dockerfile:20]: nvm, yarn is smart

Comment (garrappachc) [reply Dockerfile:20]: https://classic.yarnpkg.com/en/docs/cli/install/#toc-yarn-install-production-true-false

> Yarn will not install any package listed in devDependencies if the NODE_ENV environment variable is set to production.

**Pull request merged**